logstash-core/lib/logstash/api/lib/app/service.rb

```diff
@@ -40,4 +44,20 @@ def get(key)
     end
     LogStash::Json.dump(data)
   end
+
+  private
+
+  def has_counters?
+    return false if @snapshot.nil?
+    if @snapshot_rotation_mutex.try_lock
+      has = metric_types.include?("LogStash::Instrument::MetricType::Counter")
+      has = has &&  metric_types.include?("LogStash::Instrument::MetricType::Gauge")
+      @snapshot_rotation_mutex.unlock
+      return has
+    end
+  end
+
+  def metric_types
+    @snapshot.metric_store.all.clone.map { |t| t.class.to_s }
```

Comment:
I want to avoid messing up with probable internal references here, but yeah! not much need at the end of the day :+1: